addons/account/static/tests/tours/account_product_catalog_tests.js

```diff
@@ -0,0 +1,42 @@
+import { accountTourSteps } from "@account/js/tours/account";
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('search_read_tour', {
+    url: "/odoo",
+    steps: () => [
+    ...accountTourSteps.goToAccountMenu("Go to Invoicing"),
+    {
+        content: "Go to Customers",
+        trigger: 'span:contains("Customers")',
+        run: "click",
+    },
+    {
+        content: "Go to Invoices",
+        trigger: 'a:contains("Invoices")',
+        run: "click",
+    },
+    {
+        content: "Create new invoice",
+        trigger: '.o_control_panel_main_buttons .o_list_button_add',
+        run: "click",
+    },
```

Comment:
Can you create the invoice PY side and start the tour directly on the opened invoice ?